src/views/Expenses.js

```diff
@@ -0,0 +1,73 @@
+import { useEffect, useState } from 'react';
+import AddExpensesForm from '../components/AddExpensesForm';
+import ExpensesList from '../components/ExpensesList';
+
+const Expenses = () => {
+  const [expenses, setExpenses] = useState([]);
+  const token = process.env.REACT_APP_MOSTASH_API_KEY;
+  const baseURL = process.env.REACT_APP_MOSTASH_BASE_URL;
+  const headers = {
+    'Stash': token,
+    'Content-Type': 'application/json'
+  };
+
+  // Run on initial render only
+  useEffect(() => {
+    const fetchExpenses = async () => {
+      const url = `${baseURL}/items.json?stash=${token}&kind=expense`;
+
+      try {
+        const response = await fetch(url);
+        const expenseData = await response.json();
+        setExpenses(expenseData);
+      } catch {
+        alert('Error loading expenses. Please try again later.');
+      }
+    }
+
+    fetchExpenses();
```

Comment:
why do you define a function here? is this necessary for it to be async?

Reply:
Do you mean why define it inside `useEffect` or why define a function at all?

Reply:
i mean why define it at all. i can not infer why this is needed.

Reply:
My understanding is that `await` will only work inside of an async function.

Reply:
ah got ya. sorry, I thought we could mark the arrow function as async, but we can't. my bad.